### PR TITLE
Update fuel-test index

### PR DESCRIPTION
**Pull request by ra0x3**

### CODE REVIEW

File: docs/src/examples/block-explorer.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Block Explorer
 
-An extremely basic block explorer backend implementation that shows how to leverage basic Fuel indexer abstractions in order to build a cool dApp backend.
+An rudimentary block explorer backend implementation demonstrating how to leverage basic Fuel indexer abstractions in order to build a cool dApp backend.
```
Comment (deekerno) [inline docs/src/examples/block-explorer.md:3]: Consider:
> A rudimentary...

File: docs/src/examples/block-explorer.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 ```rust
-//! An extremely basic block explorer implementation that shows how blocks, transactions,
-//! contracts, and accounts can be persisted into the database.
+//! A rudimentary block explorer implementation demonstrating how blocks, transactions, recepits
```
Comment (deekerno) [inline docs/src/examples/block-explorer.md:6]: Consider:
> ...receipts

File: docs/src/examples/block-explorer.md [hunk 3]
```diff
@@ -28,17 +28,13 @@ An extremely basic block explorer backend implementation that shows how to lever
...
-    types::{Bytes32, ContractId},
-    utils::sha256_digest,
-};
+use fuel_indexer_plugin::{types::Bytes32, utils::sha256_digest};
 use std::collections::HashSet;
 
-// Using an ID for some abstraction (Contract, Account, etc), naively derive
-// a unique ID for some database entity
-pub fn derive_id(id: Vec<u8>, data: Vec<u8>) -> Bytes32 {
+// Entitiees require IDs - naively create unique IDs using some caller and the data used
```
Comment (deekerno) [inline docs/src/examples/block-explorer.md:34]: Consider:
> Entities...

File: examples/block-explorer/explorer-index/src/lib.rs [hunk 4]
```diff
@@ -23,17 +23,13 @@
...
-    types::{Bytes32, ContractId},
-    utils::sha256_digest,
-};
+use fuel_indexer_plugin::{types::Bytes32, utils::sha256_digest};
 use std::collections::HashSet;
 
-// Using an ID for some abstraction (Contract, Account, etc), naively derive
-// a unique ID for some database entity
-pub fn derive_id(id: Vec<u8>, data: Vec<u8>) -> Bytes32 {
+// Entitiees require IDs - naively create unique IDs using some caller and the data used
```
Comment (deekerno) [inline examples/block-explorer/explorer-index/src/lib.rs:29]: Consider:
> Entities...

**Review by deekerno — APPROVED** [review]
Approved. 👍🏽 Comments are only for spelling nits.

File: docs/src/examples/block-explorer.md (hunk 1 above)
Comment (ra0x3) [reply docs/src/examples/block-explorer.md:3]: Yea I thought that was the change that went in the docs PR? Not sure how it got changed back. Anyway, will make this change

**Review by deekerno — APPROVED** [review]
:shipit:

**Pull request merged**